frontend/packages/topology/src/actions/provider.ts

```diff
@@ -12,7 +13,8 @@ export const useTopologyWorloadActionProvider = (element: GraphElement) => {
   }, [element]);
 
   return useMemo(() => {
-    if (!isNode(element)) return [[], true, undefined];
+    if (!isNode(element) || element.getType() === TYPE_KNATIVE_REVISION)
```

Comment:
this also looks like bad dependency ....

Reply:
Fixed it